### ISSUE TITLE
Consider factoring out "bitcoins" string in AskPassphraseDialog

### ISSUE THREAD

**Issue by JeremyRand** [bitcoin-core/gui#380]
The Bitcoin-Qt GUI contains the word "bitcoins" in [two](https://github.com/bitcoin/bitcoin/blob/34d1d6a112709cada2c415deae7f7989b2d8c2f3/src/qt/askpassphrasedialog.cpp#L111) [strings](https://github.com/bitcoin/bitcoin/blob/34d1d6a112709cada2c415deae7f7989b2d8c2f3/src/qt/askpassphrasedialog.cpp#L119) in `AskPassphraseDialog`.  In the same spirit as https://github.com/bitcoin/bitcoin/pull/7192 , it would be nice to factor that out into a single constant string, so that translations are more consistent, and forks that call their coins something other than "bitcoins" can decrease their diff size relative to upstream.  Seems like maybe the `BitcoinUnits` class could be used for this, since it already [defines](https://github.com/bitcoin/bitcoin/blob/efff9c3494a944a1a9288fec13e8bb5f23235920/src/qt/bitcoinunits.cpp#L69) the string "Bitcoins".

**Comment by fanquake** [comment]
If you want to make changes that improve the code here, that's fine, but

> and forks that call their coins something other than "bitcoins" can decrease their diff size relative to upstream.

that alone is not going to be a reason for us to make changes in our codebase.

**Comment by JeremyRand** [comment]
> If you want to make changes that improve the code here, that's fine, but
> 
> > and forks that call their coins something other than "bitcoins" can decrease their diff size relative to upstream.
> 
> that alone is not going to be a reason for us to make changes in our codebase.

Yes, I know.  The primary reason I'd hope for this change to be made is just that it makes the code cleaner, as with the merged PR that I linked.

**Comment by hebasto** [comment]
> > If you want to make changes that improve the code here, that's fine, but
> > > and forks that call their coins something other than "bitcoins" can decrease their diff size relative to upstream.
> > 
> > 
> > that alone is not going to be a reason for us to make changes in our codebase.
> 
> Yes, I know. The primary reason I'd hope for this change to be made is just that it makes the code cleaner, as with the merged PR that I linked.

Closing for now. Feel free to open a PR to continue the discussion on the proposed code improvements.
